test/api.test.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+const expect = require('chai').expect;
+require('dotenv').config()
+const {apiStatus, specstoreApiUpload, apiAuthChangeUsername, apiAuthPublicKey, apiStatusWithHeaders, apiBitstoreAuthorize, apiBitstoreInfoValidToken, apiBitstoreInfoInvalidToken} = require('../scripts/api.js')
+
+
+describe('specstore service api', function () {
+  it('specstore status works fine', async function () {
+    const url = `${process.env.SPECSTORE}/${process.env.OWNERID}/${process.env.DATAPACKAGE_NAME}/1`
```

Comment:
will `1` work here?  what will happen after second push?

Reply:
I know, it is hardcoded, but this is an old test which does not include `publishing`.There is no repo for this dataset, just sample for checking